lib/index.d.ts

```diff
@@ -1,4 +1,4 @@
-import type { CookieSerializeOptions } from "@types/cookie";
+import type { CookieSerializeOptions } from "cookie";
```

Comment:
There are no types inside the package "cookie". I guess it "works" because something (TS?) will look for types in @types/cookie automatically?

If so, can you explain why we're updating this line? Thanks!

Reply:
Just saw https://github.com/vvo/next-iron-session/pull/347 (was fixed already). And read the doc here: https://www.typescriptlang.org/tsconfig#typeRoots. Thanks